### DISCUSSION TITLE
Permission to re-use some indicator menu icons

### DISCUSSION THREAD

**Discussion by sri-rang** [rxhanson/Rectangle#550] [Q&A]
Hello,

I am creating an app for Linux (X window based DEs) that is inspired by Rectangle.

The project is located here: https://gitlab.com/sri-at-gitlab/projects/xlap

I would like to reuse some of your icons, specifically the ones in the indicator menu that indicate the layout of the window.

Would that be okay?

**Comment by sri-rang** [comment]
These are the specific icons I am interested in re-using: https://github.com/rxhanson/Rectangle/tree/master/Rectangle/Assets.xcassets/WindowPositions

**Comment by rxhanson** [comment]
Sure thing, thanks for asking. If you don’t mind listing the credit somewhere in the repo or the app, I would appreciate it.

**Reply by sri-rang** [reply]
Credit and Inspiration added here: https://gitlab.com/sri-at-gitlab/projects/xlap#credits-inspiration

**Comment by sri-rang** [comment]
> Sure thing, thanks for asking. If you don’t mind listing the credit somewhere in the repo or the app, I would appreciate it.

Will do, thanks!
